src/components/logservice/Form.tsx

```diff
@@ -0,0 +1,200 @@
+                                                    max={24}
+                                                    type="number"
+                                                    value={responses["totalHours"]}
+                                                    onChange={(value) => setResponse("totalHours", value)}
+                                                />
+                                            </Grid.Col>
+                                            <Grid.Col span={4}>
+                                                <FormInput
+                                                    required
+                                                    readonly={inReview}
+                                                    headline="Service Type"
+                                                    type="dropdown"
+                                                    value={responses["type"]}
+                                                    options={["Community Service", "Capstone", "Internship", "Project", "Other"]}
+                                                    onChange={(value) => setResponse("type", value)}
+                                                />
+
+                                            </Grid.Col>
+                                        </Grid>
+                                        <div>
+                                            <FormInput
+                                                required
+                                                readonly={inReview}
+                                                headline="Tell us about your experience"
+                                                paragraph
+                                                value={responses["description"]}
+                                                onChange={(value) => setResponse("description", value)}
+                                            />
+                                        </div>
+                                    </div>
+
+                                    <div className="flex gap-x-4 w-max m-auto">
+                                        {inReview && (
+                                            <Button onClick={() => setInReview(false)} color="dark-blue" size="md"
+                                            >
+                                                Edit
+                                            </Button>
+                                        )}
+                                        <Button className={classes.continueButton} color="dark-blue" size="md">
+                                            {inReview ? "Submit" : "Continue"}
+                                        </Button>
+                                    </div>
+                                </>
+                            )}
+                        </form>
+                    </div>
+                </div>
+            </div>
+
+
+            {/* {showExitDialogue && (

```

Comment:
We should removed commented out code before merging.

Reply:
removed the commented code